examples/google/example.go

```diff
@@ -178,8 +171,8 @@ func googleCert(outputDir string, alg jwa.KeyAlgorithm, signGq bool) error {
 		return fmt.Errorf("malformatted skid: %w", err)
 	}
 
-	skidpkt, err := pktoken.FromJSON(skidDecoded)
-	if err != nil {
+	var skidpkt *pktoken.PKToken
+	if err := json.Unmarshal(skidDecoded, skidpkt); err != nil {
```

Comment:
Is this handle the error case in which the skidpkt is nil? I assume so right?

Reply:
`skidpkt` will be nil if there is an error thrown, so we check if the error was thrown.